src/main/java/com/exasol/bucketfs/testcontainers/TestcontainerBucketFactory.java

```diff
@@ -54,7 +54,7 @@ public synchronized Bucket getBucket(final String serviceName, final String buck
                     .stateRetriever(monitor.createStateRetriever()) //
                     .serviceName(serviceName) //
                     .name(bucketName) //
-                    .ipAddress(this.host) //
+                    .host(this.host) //
```

Comment:
This is a breaking change in the API. I think we need to make a new release of BFSJ with a deprecated method `ip` that delegates to `host`.